docs/src/main/sphinx/connector/hudi.md

```diff
@@ -12,8 +12,8 @@ To use the Hudi connector, you need:
 
 - Hudi version 0.12.3 or higher.
 - Network access from the Trino coordinator and workers to the Hudi storage.
-- Access to a Hive metastore service (HMS).
-- Network access from the Trino coordinator to the HMS.
+- Access to a Hive metastore service (HMS) or AWS Glue.
+- Network access from the Trino coordinator to the HMS or AWS Glue.
```

Comment:
We don't run CI for AWS Glue with Hudi connector at this time. I hesitate documenting it in this situation.

Reply:
I've tested it personally that it works.   I don't know if that helps or not.